### DISCUSSION TITLE
参考文献格式不一致

### DISCUSSION THREAD

**Discussion by monwo33866** [sikouhjw/gdutthesis-Word#13] [General]
建立新的参考文献后，格式不一致，请问作者这个怎么解决
![image](https://user-images.githubusercontent.com/60174155/162686249-8d569be8-d551-4858-89cd-3865d904975f.png)

**Comment by sikouhjw** [comment]
参考文献我在 https://github.com/sikouhjw/gdutthesis-Word/commit/1aaed93dcfc53c7f7138f83ab0d2d69559fee65e 重新设置了一下，因为发现之前有些地方没设置好。

可以参考《参考文献样式悬挂缩进》[^1]，重新设置一下样式，然后选中参考文献，点击样式刷新格式。

[^1]: https://github.com/sikouhjw/gdutthesis-Word/wiki#%E5%8F%82%E8%80%83%E6%96%87%E7%8C%AE%E6%A0%B7%E5%BC%8F%E6%82%AC%E6%8C%82%E7%BC%A9%E8%BF%9B

**Reply by monwo33866** [reply]
谢谢学长，已解决
